src/app.js

```diff
@@ -151,11 +178,11 @@ export function app(props, container) {
       var oldValue = i === "value" || i === "checked" ? element[i] : oldProps[i]
 
       if (value !== oldValue) {
-        setElementProp(element, i, value, oldValue)
+        value !== oldValue && setElementProp(element, i, value, oldValue)
```

Comment:
Double checking for `value !== oldValue` this was perviously fixed by https://github.com/hyperapp/hyperapp/pull/429

Reply:
😅 Whoops. Thanks!